makeit/mconfig

```diff
@@ -21,7 +21,7 @@ tgtcxx_opts=$hstcxx_opts
 hststatic=0
 tgtstatic=0
 appsec=0
-package_version=`git describe --match 'v[0-9]*' --dirty --always 2>/dev/null || cat VERSION 2>/dev/null || echo "" | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"`
+package_version=`(git describe --match 'v[0-9]*' --dirty --always 2>/dev/null || cat VERSION 2>/dev/null || echo "") | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"`
```

Comment:
So ... Do you know what this is _supposed_ to be doing?

```
$ (git describe --match 'v[0-9]*' --dirty --always 2>/dev/null || cat VERSION 2>/dev/null || echo "") | sed -e "s/^v//;s/-/_/;s/_/-/;s/_/./g"
3.0.0-50-gfa6c9ef-dirty
```

So, it's removing the leading 'v' .. .and that's it.

Reply:
You didn't do a direct copy/paste.  You're missing a 'g':

```
$ (git describe --match 'v[0-9]*' --dirty --always 2>/dev/null || cat VERSION 2>/dev/null || echo "") | sed -e "s/^v//;s/-/_/g;s/_/-/;s/_/./g"
3.0.0-53.gaadaa34
```
It is supposed to turn all dashes to underscore, then the first underscore back to dash, and the rest of the underscores to dot.

Reply:
:+1: